### PR TITLE
Run first scenario

### PR DESCRIPTION
**Pull request by anciuvaga**
- Run first scenario

### CODE REVIEW

File: src/test/java/cucumber_runner/TestRunner.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @CucumberOptions(
         glue = { "stepDefinition", "hooks" },
         features = { "src/test/resources/features" },
-        tags = { "not @amiordan", "not @mraicov", "not @aciuvaga" },
+        tags = { "@Andrei_Ciuvaga" },
```
Comment (CristianManic) [inline src/test/java/cucumber_runner/TestRunner.java]: revert this change and edit just your tag with `not` prefix as others don't need to run  your scenarios

File: src/test/resources/features/wishlistFunctionality.feature [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#  Scenario: Add a product from WishList to Cart
+#    When User clicks on a "Electronics"category from category bar
+#    And User clicks on "Cell phones" subcategory
+#    And User clicks on "HTC One Mini Blue" product
+#    And User clicks on "Add to wishlist" button
+#    And User navigates to "Wishlist" page
+#    Then Product "HTC One Mini Blue" is displayed in the WishList
+#    When User checks "Add toCart" checkbox
+#    And presses "ADD TO CART" button
+#    Then selected product from "Wishlist" is added successfully to ShoppingCart
```
Comment (CristianManic) [inline src/test/resources/features/wishlistFunctionality.feature]: please don't leave comments

File: src/test/resources/features/wishlistFunctionality.feature [hunk 3]
```diff
@@ -0,0 +1,77 @@
+
```
Comment (CristianManic) [inline src/test/resources/features/wishlistFunctionality.feature]: delete blank space and rename feature file as specified in PR #12

File: src/test/java/cucumber_runner/TestRunner.java (hunk 1 above)
Comment (anciuvaga) [reply src/test/java/cucumber_runner/TestRunner.java]: Done

File: src/test/resources/features/wishlistFunctionality.feature (hunk 2 above)
Comment (anciuvaga) [reply src/test/resources/features/wishlistFunctionality.feature]: Done

File: src/test/resources/features/wishlistFunctionality.feature (hunk 3 above)
Comment (anciuvaga) [reply src/test/resources/features/wishlistFunctionality.feature]: Done

File: src/test/java/hooks/TestHooks.java [hunk 4]
```diff
@@ -2,24 +2,19 @@
...
 public class TestHooks extends AbstractStepDef {
-
-    private ConfigFileReader fileReader = new ConfigFileReader();
-
     private WebDriverManager webDriverManager = new WebDriverManager();
 
     @Before
     public void launchChromeDriver() {
-        webDriver = webDriverManager.getDriver();
+        webDriver = this.webDriverManager.getDriver();
```
Comment (vdones) [inline src/test/java/hooks/TestHooks.java]: why should you use the keyword "this" in this method?

File: src/test/java/hooks/TestHooks.java [hunk 5]
```diff
@@ -2,24 +2,19 @@
...
     }
 
     @After
     public void afterScenario() {
-        webDriverManager.closeDriver();
+        this.webDriverManager.closeDriver();
```
Comment (vdones) [inline src/test/java/hooks/TestHooks.java]: why should you use the keyword "this" in this method?

File: src/test/java/stepDefinition/AbstractStepDef.java [hunk 6]
```diff
@@ -1,8 +1,19 @@
...
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 public abstract class AbstractStepDef {
-
     protected static WebDriver webDriver;
+    protected ConfigFileReader fileReader = new ConfigFileReader();
+    WebDriverWait wait;
```
Comment (vdones) [inline src/test/java/stepDefinition/AbstractStepDef.java]: you should restrict access to variables

**Review by vdones — CHANGES_REQUESTED** [review]

File: src/test/java/hooks/TestHooks.java (hunk 4 above)
Comment (anciuvaga) [reply src/test/java/hooks/TestHooks.java]: Done

File: src/test/java/hooks/TestHooks.java (hunk 5 above)
Comment (anciuvaga) [reply src/test/java/hooks/TestHooks.java]: Done

File: src/test/java/stepDefinition/AbstractStepDef.java (hunk 6 above)
Comment (anciuvaga) [reply src/test/java/stepDefinition/AbstractStepDef.java]: Done

**Review by CristianManic — APPROVED** [review]
